test/data/testvib.py

```diff
@@ -130,7 +130,7 @@ def testenthalpy(self):
 
     def testentropy(self):
          """Is the entropy reasonable"""
-         self.assertAlmostEqual(0.03601749, self.data.entropy, self.entropy_places)
+         self.assertAlmostEqual(0.00012080325339594164, self.data.entropy, self.entropy_places)

```

Comment:
Do you know what the units here are? It is kcal/(mol K) like the logfile suggests?

Reply:
True. I insisted a bit in hartree/(particle*kelvin) to keep consistency with other fields in the docs such as `enthalpy` and `freeenergy`, but we can change them all. Essentially, "particle" here means "unity".

Reply:
SGTM